doc/source/extending-jupyterhub.rst

```diff
@@ -81,9 +81,10 @@ configure JupyterHub to build off of this image.
 
    .. code-block:: bash
 
-          numpy=1.12.1
-          scipy=0.19.0
-          matplotlib=2.0
+          pip==9.0.1
```

Comment:
it'll have an old version though. It uses pip-8.1.2. If you think it's better not to mess with it, alright.